src/main/java/ca/bc/hlth/mohorganizations/LoggingFilter.java

```diff
@@ -0,0 +1,46 @@
+package ca.bc.hlth.mohorganizations;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+import org.springframework.web.util.ContentCachingResponseWrapper;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+@Component
+public class LoggingFilter extends OncePerRequestFilter {
+    private static final Logger LOGGER = LoggerFactory.getLogger(LoggingFilter.class);
+
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain) throws ServletException, IOException {
+
+        LOGGER.info("{} : {}", request.getMethod(), request.getRequestURI());
+        ContentCachingResponseWrapper cachedResponse = new ContentCachingResponseWrapper(response);
+
+        filterChain.doFilter(request, cachedResponse);
+
+        int responseStatus = cachedResponse.getStatus();
+
+
+        if (!isSuccessfulRequest(responseStatus)) {
+            byte[] responseBody = cachedResponse.getContentAsByteArray();
+            LOGGER.warn("Completed : {}", responseStatus);
```

Comment:
There's two log statements with the same content (one at level warn and the other info). I would be inclined to just use a single INFO statement before the if block. That being said, if you want to separate the two you could maybe make it clearer that one is an error scenario. E.g. "Error " vs "Completed" since "Completed" might be a little misleading if the request didn't complete normally.

I would also tend to use LOGGER.error in this scenario instead of .warn as this really shouldn't happen under normal operation and would definitely be an error scenario.